cli/src/deploy/mod.rs

```diff
@@ -10,8 +10,8 @@ use std::io::BufReader;
 use std::path::Path;
 
 use ascii_table::{AsciiTable, Column};
-use log::error;
 
+use crate::error;
```

Comment:
if you do something like this in `lib.rs`, you won't have to import the macro in every mod you use it in:
```rs
#[macro_use]
mod logger;
```